{"name": "test_update_user_with_auth", "status": "failed", "statusDetails": {"message": "AssertionError: assert 403 == 200\n +  where 403 = <Response [403]>.status_code", "trace": "self = <test_update_user.TestUpdateUser object at 0x11895ea80>\n\n    def test_update_user_with_auth(self):\n        user_data = {\"name\": \"Updated User\"}\n        headers = {\"Authorization\": \"Bearer <accessToken>\"}\n        response = requests.patch(f\"{BASE_URL}/auth/user\", json=user_data, headers=headers)\n>       assert response.status_code == 200  # Успешное обновление\nE       assert 403 == 200\nE        +  where 403 = <Response [403]>.status_code\n\ntests/test_update_user.py:11: AssertionError"}, "start": 1732752226573, "stop": 1732752226733, "uuid": "b7260369-6ba4-44bd-a8d0-8b3936ed2881", "historyId": "ab27f8bb394885cc72f7183d7134c8bd", "testCaseId": "ab27f8bb394885cc72f7183d7134c8bd", "fullName": "tests.test_update_user.TestUpdateUser#test_update_user_with_auth", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_update_user"}, {"name": "subSuite", "value": "TestUpdateUser"}, {"name": "host", "value": "admins-MacBook-Pro.local"}, {"name": "thread", "value": "23420-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_update_user"}]}